### PR TITLE
Add markdown input style for user 

### PR DESCRIPTION
**Pull request by pabloemilio97**
* Markdown style for messages and about me page
* Data is sanitized using Jsoup: no HTML is permitted as input.

### CODE REVIEW

**Review by osjuga — APPROVED** [review]
Looks good!

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 1]
```diff
@@ -12,44 +12,47 @@
 import com.google.codeu.data.User;
 import org.jsoup.Jsoup;
 import org.jsoup.safety.Whitelist;
+import org.commonmark.node.*;
```
Comment (gauravmishra) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: suggestion:  you should avoid wildcard imports in order (1) not to bloat the application, (2) to let the reader know exactly what classes you're using.

See https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 2]
```diff
@@ -12,44 +12,47 @@
...
+import org.commonmark.parser.Parser;
+import org.commonmark.renderer.html.HtmlRenderer;
 
 /**  Handles fetching and saving user data. */
 
 @WebServlet("/about")
 public class AboutMeServlet extends HttpServlet {
-  
+
```
Comment (gauravmishra) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: please remove unnecessary whitespaces in this file. These add lines to the diff where there no changes, which is confusing (you can use an automatic formatter to do this) . Ditto below.

**Review by gauravmishra — APPROVED** [review]
Great work! Left a few nits.

**Pull request merged**